tests/tests/test-fees/test-fee-multiplier.ts

```diff
@@ -425,3 +426,34 @@ describeDevMoonbeam("Fee Multiplier - XCM Executions", (context) => {
     expect(initialValue.eq(postValue), "Fee Multiplier has changed between blocks").to.be.true;
   });
 });
+
+describeDevMoonbeam("TransactionPayment Runtime Queries", (context) => {
+  it("should be able to query length fee", async function () {
+    const adjusted_length_fee =
+      await context.polkadotApi.call.transactionPaymentApi.queryLengthToFee(1n);
+    expect(adjusted_length_fee.toBigInt()).to.eq(1_000_000_001n);
```

Comment:
What is the significance of this number `1_000_000_001` ?
Is it "1_000_000_000 + 1" ? if so a small text as the error reason should suffice in helping future generations understand what this test means
e.g. `expect(adjusted_length_fee.toBigInt(),"Length fee doesn't match <INSERT_CONST_NAME_HERE> and plus 1 for byte").to.eq(1_000_000_001n);`

Reply:
Added an explanation in 11293fe. The point of the test was just to show that the query works, but I might as well explain the algo :shrug: